### PR TITLE
Add quinary support and motorola-ocean

### PR DESCRIPTION
**Pull request by Ultracoolguy**
I'm not sure if the name is completely accurate since it could theoretically work on other devices non-msm8953, but I don't have such devices so I can't test.

### CODE REVIEW

**Comment by Minecrell** [comment]
I wonder if we can make like a variable (similar to the `WcdPlaybackPCM`) and set that to `QUIN_MI2S_RX` or `QUAT_MI2S_RX`, depending on the device.

**Comment by Ultracoolguy** [comment]
Changed method of using Quinary.

File: ucm2/platforms/msm8916/qdsp6-components.conf [hunk 1]
```diff
@@ -1,4 +1,13 @@
 If.qdsp6 {
+	Condition {
+		Type String
+		Haystack "${var:IsQuinary}"
+		Needle "yes"
+	}
```
Comment (Minecrell) [inline ucm2/platforms/msm8916/qdsp6-components.conf:6]: Perhaps make this a bit easier

```
Condition {
	Type String
	Empty "${var:MultiMedia3Port}"
}
True.Define.MultiMedia3Port "QUAT_MI2S_RX"
```

then in ocean HiFi

```
Define {
	...
	MultiMedia3Port "QUIN_MI2S_RX"
}
```

Would this work?

Comment (Ultracoolguy) [reply ucm2/platforms/msm8916/qdsp6-components.conf]: Done.

File: ucm2/platforms/msm8916/qdsp6-components.conf [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
-		Type String
-		Haystack "${CardComponents}"
-		Needle "qdsp6"
-	}
-
-	True {
-		<platforms/msm8916/qdsp6.conf>
-	}
+
+        Condition {
```
Comment (Minecrell) [inline ucm2/platforms/msm8916/qdsp6-components.conf:3]: I guess I'd prefer if this stays as tabs instead of spaces

Comment (Ultracoolguy) [reply ucm2/platforms/msm8916/qdsp6-components.conf]: This is what I get for copypasting from terminal instead of doing properly.

**Pull request merged**